locustempus/templates/404.html

```diff
@@ -3,15 +3,32 @@
 {% block title %}Page Not Found{% endblock %}
 
 {% block content %}
-<div class="row justify-content-md-center">
-    <div class="col-md-auto text-center my-5">
-        <h2>404: Page Not Found</h2>
+<div class="http-error-prompt row">
+    <div class="col-12 col-lg-5 pt-5">
+        <div class="http-error-prompt__decor text-center">
+            <img src="{{STATIC_URL}}img/prompt-404.svg" />
+        </div>
+    </div>
+    <div class="col-12 col-lg-7 text-center text-lg-left">
+        <h2>Error 404: Page not found</h2>
 
         <p>
           We're sorry. The page you're looking for doesn't exist at this address.
```

Comment:
@zmustapha At some point, we should come up with something pithy to say about needing a map when you get lost.

Reply:
I have 4 days this weekend to think about it, hahahahhaha